portal/eproms/views.py

```diff
@@ -139,7 +139,19 @@ def privacy():
 def terms_and_conditions():
     """ terms-and-conditions of use page"""
     user = current_user()
-    terms = VersionedResource(app_text(Terms_ATMA.name_key()))
+    if user:
```

Comment:
Should this logic be moved to the `VersionedResource` model? Seems it would be useful elsewhere (than this one view)

Reply:
It kind of is (in that `Terms_ATMA()` returns one thing if `kwargs.get('organization') and kwargs.get('role')`, and another if not), but I don't think we want the actual org and role lookup logic in there...at least, it seems like Paul's been intentionally avoiding any sort of actual object query logic in those AppTextModelAdapters

Reply:
Sorry I wasn't implying something as big as moving the org logic into any other model; you correctly guessed my vague suggestion in your other comment :)